client/src/_reactComponents/PanelHeaderComponents/ContentCard.jsx

```diff
@@ -25,23 +25,23 @@ export default function ContentCard({
   isAssigned,
   isFolder,
   isPublic,
-  name,
-  fullName,
+  title,
+  ownerName,
   menuItems,
   suppressAvatar = false,
-  editable = false,
-  autoFocusName = false,
-  showStatus = true,
+  showOwnerName = true,
+  editableTitle = false,
+  autoFocusTitle = false,
 }) {
   if (!imagePath) {
     imagePath = "/activity_default.jpg";
   }
-  const [cardName, setCardName] = useState(name);
+  const [cardTitle, setCardTitle] = useState(title);
```

Comment:
Because you have have created this editable title, you have a separate state variable `cardTitle` that does not change with the `title` prop. This means, if the title changes from the outside (e.g., with the Setting panel), the title will not change. If the `title` prop changes, that means it was changed from outside and you need to set `cardTitle` to match. To do this, you can use a `useEffect` hook to run code whenever `title` changes, where `useEffect` has to be imported from `React`:
```
  useEffect(() => {
    setCardTitle(title);
  }, [title]);
```